lib/usb/usb_fx07_common.c

```diff
@@ -304,23 +321,6 @@ void stm32fx07_poll(usbd_device *usbd_dev)
 		usbd_dev->rxbcnt = 0;
 	}
 
-	/*
-	 * There is no global interrupt flag for transmit complete.
-	 * The XFRC bit must be checked in each OTG_DIEPINT(x).
-	 */
-	for (i = 0; i < 4; i++) { /* Iterate over endpoints. */
```

Comment:
ok, existing bug then :)  think it might have already been filed elsewhere, but fine for now then.